rock-paper-scissors/README.md

```diff
@@ -0,0 +1,37 @@
+# Rock-Paper-Scissors
```

Comment:
This README file hopefully makes everything clear, insofar as my design decisions.